doc/source/ray-observability/ray-metrics.rst

```diff
@@ -123,3 +117,14 @@ If you open this in the browser, you should see the following output:
   ray_num_requests_total{Component="core_worker",Version="3.0.0.dev0",actor_name="my_actor"} 2.0
 
 Please see :ref:`ray.util.metrics <custom-metric-api-ref>` for more details.
+
+Customize prometheus export port
```

Comment:
@rkooo567 , I renamed this from "Export Metrics from Prometheus Endpoint"

Let me know if I'm interpreting these docs the right way and the new name is appropriate.

Reply:
yeah this looks good. The purpose of the section is for people who don't want to use the file-based service discovery.

Reply:
maybe we can write one sentence like `Ray by default provides the service discovery file, but you can directly scrape metrics from prometheus ports`